### PR TITLE
Adds homepage

### PR DESCRIPTION
**Pull request by QuitHub**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Seyda's Bakes</title>
```
Comment (neutrino49910) [inline index.html:4]: Great title bro

File: index.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    </head>
+    <body>
+        <h1>Seyda's Bakes<h1>
+    </body>
```
Comment (aweissaeed) [inline index.html:8]: stick to your day job

File: index.html (hunk 1 above)
Comment (neutrino49910) [reply index.html:4]: why is the body so far down ???

File: index.html (hunk 2 above)
Comment (QuitHub) [reply index.html:8]: go fuck yourself

File: index.html (hunk 1 above)
Comment (QuitHub) [reply index.html:4]: Dunno about you, but my body is below my head....

**Pull request merged**